src/containers/shared/components/Transaction/XChainAddAttestation/parser.ts

```diff
@@ -0,0 +1,40 @@
+import { TransactionParser } from '../types'
+import { XChainAddAttestation, XChainAddAttestationInstructions } from './types'
+
+export const parser: TransactionParser<
+  XChainAddAttestation,
+  XChainAddAttestationInstructions
+> = (tx) => {
+  // TODO: get bridge owner somehow
+  // it's not necessarily in the metadata
+  const claimAttestations =
+    tx.XChainAttestationBatch.XChainClaimAttestationBatch.map(
+      (attestation) => ({
+        send: attestation.XChainClaimAttestationBatchElement.Amount,
```

Comment:
Why do we call this field `send` for this file, when the actual field is the `Amount`? (Should the naming match?)

Reply:
It's called `send` for payment transactions, we're using the same paradigm here.